resources/views/fields/radio.blade.php

```diff
@@ -1,25 +1,34 @@
-<div class="{{ $field->divClass ?? config('livewire-forms.defaults.divClass') }} required">
-    @include('livewire-forms::fields.gdpr')
+<div @class([
+    $field->divClass ?? config('livewire-forms.defaults.divClass'),
+    $field->colClass ?? config('livewire-forms.defaults.colClass')])
+>
+    @include('livewire-forms::fields.label')
 
-    @foreach ($field->options as $key => $value)
-        <div class="form-group custom-control custom-radio">
-            <input
-                @include('livewire-forms::fields.binding')
-                type="radio"
-                class="{{ $field->class }}"
-                id="{{ $field->getName() . '.' . $loop->index }}"
-                name="{{ $field->getName() }}"
-                value="{{ $field->useValueAsKeys ? $value : $key }}"
-                @if ($field->dusk) dusk={{ $field->dusk }} @endif
-            >
-            <label
-                class="{{$field->labelClass}} select-none"
-                for="{{ $field->getName() . '.' . $loop->index }}"
-            >
-                {{ $value }}
-            </label>
-        </div>
-    @endforeach
+    <div class="d-flex flex-column gap-2">
```

Comment:
Misschien moeten we de gap-2 ook in de config plaatsen zodat deze overschreven kan worden indien nodig?